src/UI/CompletionsView.re

```diff
@@ -85,17 +155,60 @@ let createElement =
               |> int_of_float;
             let height =
               Types.EditorFont.getHeight(editorFont) +. 0.5 |> int_of_float;
+            let remainingWidth = 400 - width;
+
+            let detailElem =
+              switch (curr.completionDetail) {
+              | None => React.empty
+              | Some(v) when String.length(v) > 0 =>
```

Comment:
If you call `v` here `text` instead, it's both more descriptive and you can use punning with both `measure(~text, ...)` and `<Text text ... />` later.

Reply:
Nice!